app/src/main/java/com/example/myapplication/mvp/presenter/CalculatorPresenter.kt

```diff
@@ -56,4 +57,26 @@ class CalculatorPresenter(val model: CalculatorModel, val view: CalculatorView)
         model.setOperatorTwo(EMPTY_STRING)
         model.setOperand(EMPTY_STRING)
     }
+
+    fun onClearPressed() {
+        if (model.getOperatorTwo() != EMPTY_STRING) {
```

Comment:
please use the kt functions 
model.getOperatorTwo().isNotEmpty()

Reply:
done